Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -745,14 +761,38 @@ private static DateTime setEndValidTime(DateTime endValidTimeArg, DateTime issue
             attribute = OpenSAML2ComponentBuilder.getInstance().createNPIAttribute(npi);
 
             statements
-                .addAll(OpenSAML2ComponentBuilder.getInstance().createAttributeStatement(Arrays.asList(attribute)));
+                    .addAll(OpenSAML2ComponentBuilder.getInstance().createAttributeStatement(Arrays.asList(attribute)));
         } else {
             LOG.warn("npi is missing");
         }
         return statements;
 
     }
 
+    protected List<AttributeStatement> createAcpAttributeStatements(CallbackProperties properties) {
+        final List<AttributeStatement> statements = new ArrayList<>();
+        Attribute acpAttribute;
+        Attribute iacpAttribute;
+
+        final String acp = properties.getAcpAttribute();
+        final String iacp = properties.getIacpAttribute();
+        if (NullChecker.isNotNullish(acp)) {
```

Comment:
Think StringUtils.isNotEmpty is a better check here. NullChecker.isNotNullish does not take intoaccount whitespaces string.